### ISSUE TITLE
Variable ${command:cmake.launchTargetPath} as a launch target command causes an exception.

### ISSUE THREAD

**Issue by sialiuk** [microsoft/vscode-cmake-tools#1036]
"configurations": [
        {
            "name": "(Windows) Launch",
            "type": "cppvsdbg",
            "request": "launch",
            "program": "${command:cmake.launchTargetPath}",
        },
]
With this configuration after debugger launching (F5 button) next exception is thrown:

Unhandled exception: Unhandled Promise rejection: launchTargetPath TypeError: Cannot read property 'fsPath' of undefined {} 

- Operating System: Windows
- CMake Version: 3.16.*
- VSCode Version:  1.41.1
- CMake Tools Extension Version:  1.3.0-beta
- Compiler/Toolchain: Visual C++ 2017

**Comment by bobbrow** [comment]
This was fixed in 1.3.0

**Comment by pgab** [comment]
So for me this seems not work anymore:

Operating System: Ubuntu 20.04
CMake Version: 3.16.3
VSCode Version: 1.50.0
CMake Tools Extension Version: 1.4.2
Compiler/Toolchain: GCC 9.2.1

```
{
            "name": "(gdb) Launch",
            "type": "cppdbg",
            "request": "launch",
            "program": "${command:cmake.launchTargetPath}",
            "args": [],
            "stopAtEntry": false,
            "cwd": "${workspaceFolder}",
            "environment": [
                {
                    // add the directory where our target was built to the PATHs
                    // it gets resolved by CMake Tools:
                    "name": "PATH",
                    "value": "$PATH:${command:cmake.launchTargetDirectory}"
                }
            ],
            "externalConsole": false,
            "MIMode": "gdb",
            "setupCommands": [
                {
                    "description": "Enable pretty-printing for gdb",
                    "text": "-enable-pretty-printing",
                    "ignoreFailures": true
                }
            ],
            "preLaunchTask": "${command:cmake.build}"
        },
```
